### ISSUE TITLE
Simulation.GenerateRandomWalk cannot handle start_pos != None

### ISSUE THREAD

**Issue by MonaNis** [RonnyFoerster/NanoObjectDetection#32]
Some matrix size mismatch, apparently... I tried to solve it, but so far it does not do what it should :/

**Comment by RonnyFoerster** [comment]
Can you be a bit more specific?

I am currently working on the random walk

**Comment by MonaNis** [comment]
Does the random walk function work for you when you want to have start positions that are not (0,0) ? If yes, what exactly do you pass to the function?
(I haven't tried for a while - but this was the problem when I opened this issue)

**Comment by RonnyFoerster** [comment]
keep in mind that simulations run in 1D only currently, because we dont need y anyway.

**Comment by MonaNis** [comment]
I disagree that we do not need y in general - it depends what your simulation is aiming at. I thought about repeating your simulations from last year to estimate the optimal particle concentration for the settings I'm currently using experimentally. It's more realistic to include y in there, isn't it?

**Comment by RonnyFoerster** [comment]
Can you build it as parameter in?
